mmdet/datasets/openimages.py

```diff
@@ -786,8 +825,8 @@ def get_relation_matrix(self, hierarchy_file):
             relationship between the parent class and the child class,
             of shape (class_num, class_num).
         """
-
-        assert hierarchy_file.endswith('np')
+        if self.assert_suffix:
```

Comment:
We do not need to use this option. The files are generated by us, we should always check them.

Reply:
That means we should remove all the `self.assert_suffix`?

Reply:
Yes, we should check it by default.